### PR TITLE
DA-843 Require prefix for biobank ID.

### PR DESCRIPTION
**Pull request by Seeker1911**

### CODE REVIEW

**Comment by Seeker1911** [comment]
Whitespace fixes and (the pertinent bit here) require an environment prefix when passing in biobank id parameter in participant summary.

File: rest-api/dao/participant_summary_dao.py [hunk 1]
```diff
@@ -315,6 +315,9 @@ def _add_order_by(self, query, order_by, field_names, fields):
 
   def make_query_filter(self, field_name, value):
     """Handle HPO and code values when parsing filter values."""
+    from model.config_utils import from_client_biobank_id
```
Comment (calbach) [inline rest-api/dao/participant_summary_dao.py]: Is this a datastore dependency? Is that why you aren't importing normally? If so, please just add a comment here.

File: rest-api/dao/participant_summary_dao.py [hunk 2]
```diff
@@ -315,6 +315,9 @@ def _add_order_by(self, query, order_by, field_names, fields):
...
+    if field_name == 'biobankId':
+      value = from_client_biobank_id(value)
```
Comment (calbach) [inline rest-api/dao/participant_summary_dao.py]: This is a breaking change - are you confident that no one was using this capability up until now? If you're not sure - a safer thing to do here would be to try/catch this and just log a warning if you see it (continue with the previous behavior if you see an exception).

File: rest-api/dao/participant_summary_dao.py (hunk 1 above)
Comment (Seeker1911) [reply rest-api/dao/participant_summary_dao.py]: Nope, I was experimenting and forgot to move it. It's moved now 😄 
Also, shouldn't our linters catch that ?

File: rest-api/dao/participant_summary_dao.py (hunk 2 above)
Comment (Seeker1911) [reply rest-api/dao/participant_summary_dao.py]: I believe it's not in use, HPRO pointed out the error and I've asked PTSC if they are passing in BiobankId anywhere. That being said, I think best practice would be to "deprecate it" for a while and make it a hard error in the near future. I'll make the todo for that and switch it. Thanks.

**Pull request merged**